changelog.md

```diff
@@ -7,6 +7,12 @@
 - The ndc-postgres-cli `initialize` command will now generate
   the jsonschema of the configuration format as well.
   ([#361](https://github.com/hasura/ndc-postgres/pull/361))
+- The ndc-postgres-cli `update` command will check that the configuration file has
+  not changed in the middle of the introspection process, and will retry if it did.
+  ([#362](https://github.com/hasura/ndc-postgres/pull/362))
+- The ndc-postgres-cli `update` command will not write the introspection result to
+  file if the configuration file is already up to date.
+  ([#362](https://github.com/hasura/ndc-postgres/pull/362))
```

Comment:
Consider merging these entries. It's a bit weird to have two changelog entries for one PR.